packages/data/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 4.1.0 (Unreleased)
+
+### New Feature
+
+- `withDispatch`'s `mapDispatchToProps` function takes the `registry` object as the 3rd param ([#11851](https://github.com/WordPress/gutenberg/pull/11851)).
```

Comment:
Out of curiosity, since one of the main purported benefits was to align with the signature of `withSelect`, would we want to go ahead and do that now?

Reply:
@youknowriad - I think it is your call. I can add it.

Reply:
Let's do that 🤷‍♂️

Reply:
Added in d9925fe. I have a really hard time finding a legitimate use case to add unit tests, any ideas?